docs/ecosystem.rst

```diff
@@ -0,0 +1,12 @@
+Ecosystem
+=========
+
+Here is a list of known projects, packages and integrations using pint.
+
+
+Pint packages:
+------------------
+
+- `pint <https://github.com/hgrecco/pint>` Base package
+- `pint-pandas <https://github.com/hgrecco/pint-pandas>`_ Pandas integration
+- `pint-xarray <https://github.com/xarray-contrib/pint-xarray>`_ Xarray integration
```

Comment:
- `openscm-units <https://github.com/openscm/openscm-units>` units related to simple climate modelling
- `iam_units <https://github.com/IAMconsortium/units>` Pint-compatible definitions of energy, climate, and related units to supplement the SI and other units included in Pint's [default_en.txt](https://github.com/hgrecco/pint/blob/master/pint/default_en.txt)